requirements.txt

```diff
@@ -0,0 +1 @@
+shtab==1.5.7
```

Comment:
I had to add `python-dotenv` here to get the `build` command to run successfully since it is used in the `vision-zero` program

Reply:
Sure enough - great catch. Patched -- thanks!